main.py

```diff
@@ -1,7 +1,7 @@
 import sys
 from adagrams.ui_helper import *
 from adagrams.game import draw_letters, uses_available_letters, score_word, get_highest_word_score
-
+#just adding a comment into this part
```

Comment:
[nit] You can come in and remove these spurious changes later. Same for the added line breaks in your tests.